docs/.vuepress/public/presets/toml/jetpack.toml

```diff
@@ -0,0 +1,291 @@
+home_symbol = "⌂"
+truncation_length = 2
+truncation_symbol = "▦ "
+read_only = " ■"
+use_os_path_sep = true
+style = "italic blue"
+format = ' [$path]($style)[$read_only]($read_only_style)'
+
+[cmd_duration]
+min_time = 500
+format = "[$duration ](italic bright-yellow)"
+
+[jobs]
+format = "[ $symbol$number]($style)"
+style = "white"
+symbol = "[▶ ](blue italic)"
+
+[localip]
+ssh_only = true
+format = " ◯[$localipv4](bold magenta)"
+disabled = false
+
+[time]
+disabled = false
+format = "[ $time]($style)"
+time_format = "%R"
+utc_time_offset = "local"
+style = "dimmed white"
+# time_range = "00:00:00-07:00:00"
+
+[battery]
+format = "[ $percentage $symbol]($style)"
+full_symbol = "[█](italic green)"
+charging_symbol = "[↑](italic green)"
+discharging_symbol = "[↓](italic)"
+unknown_symbol = "[░](italic)"
+empty_symbol = "[▃](italic red)"
+
+[[battery.display]]
+threshold = 40
+style = "dimmed yellow"
+
+[[battery.display]]
+threshold = 70
+style = "dimmed white"
+
+[git_branch]
+format = "[ $symbol $branch(:$remote_branch)]($style)"
+symbol = "[◬](bold bright-blue)"
+style = "bold italic bright-blue"
+
+[git_status]
+style = "italic bright-blue"
+format = """([⎪$ahead_behind$staged$modified$untracked$renamed$deleted$conflicted$stashed⎥]($style))"""
+conflicted = "[◪◦](italic bright-magenta)"
+ahead = "[▲│[${count}](bold white)│](italic green)"
+behind = "[▽│[${count}](bold white)│](italic red)"
+diverged = "[◇ ▲┤[${ahead_count}](regular white)│▽┤[${behind_count}](regular white)│](italic bright-magenta)"
+untracked = "[◌◦](italic bright-yellow)"
+stashed = "[◦◫◦](italic white)"
+modified = "[●◦](italic yellow)"
+staged = "[■┤[$count](bold white)│](italic bright-cyan)"
+renamed = "[◎◦](italic bright-blue)"
+deleted = "[✕](italic red)"
+
+[deno]
+format = " deno [∫ $version](blue italic)"
+version_format = "${major}.${minor}"
+
+[lua]
+format = " lua [${symbol}${version}]($style)"
+symbol = "⨀ " # ⏾◕
+# version_format = "${major}.${minor}"
+style = "italic bright-yellow"
+
+[nodejs]
+format = " node [◫ ($version)](italic bright-green)"
+detect_files = ["package-lock.json", "yarn.lock"]
+version_format = "${major}.${minor}"
+# detect_folders = ["node_modules"]
+# detect_extensions = ["js", "mjs", "cjs", "ts", "mts", "cts"]
+
+[python]
+format = " py [${symbol}${version}]($style)"
+symbol = "[⌉](italic bright-blue)⌊ "
+version_format = "${major}.${minor}"
+style = "italic bright-yellow"
+
+[ruby]
+format = " rb [${symbol}${version}]($style)"
+symbol = "◆ "
+version_format = "${major}.${minor}"
+style = "italic red"
+
+[rust]
+format = " rs [$symbol$version]($style)"
+symbol = "⊃ "
+version_format = "${major}.${minor}"
+style = "italic red"
+
+[package]
+format = " pkg [$symbol$version]($style)"
+version_format = "${major}.${minor}"
+symbol = "◫ "
+style = "bright-yellow italic"
+
+[swift]
+format = " sw [${symbol}${version}]($style)"
+symbol = "◁ "
+style = "italic bright-red"
+version_format = "${major}.${minor}"
+
+[aws]
+disabled = true
```

Comment:
If you're providing theming for the module, I don't see why it should be disabled if it's usually enabled by default.